pages/vi/profiles/VishalSundarrajan.md

```diff
@@ -0,0 +1,38 @@
+# Vishal Sundarrajan
+
+:round_pushpin: : Lowell, MA 01854, USA (:us:)
+
+:alarm_clock:   : Eastern Standard Time (EST)     
+
+:computer:      : macOS High Sierra (Version: 10.13.6) 
+*************
+
+## Who am I ?
+
+&nbsp; An excited and inspiring software programmer who simply enjoy contributing open source applications and creating my own 
+designs from scratch. I code in multiple languages yet try hard to maintain the objective of solving a problem in an optimal way. 
+
+> *I learn to grow and grow to learn*
+
+## What technologies do I know?
+
++ **Backend** : C, C++, Ruby, Java 8, Python, C#, .Net, Node.js, Tcl, Linux Shell Scripting
++ **Frontend** : HTML5, CSS, JavaScript (Angular.js, React.js)
++ **Database** : SQL, Mongodb, NoSQL
++ **Cloud** : AWS (EC2, MariaDB), Git, Docker, Google Firebase
++ **Tools/Frameworks** : Django, Spring boot, Spring, Hibernate, Servlet, Catch2, googletest, Maven, pip
+
+
+## What are my hobbies?
+
+1. If I have enough time and food, I play chess all day.     
+
+    -Check out my elo [here](https://www.chess.com/member/vishalicious) and send me a game request if you think you are good. :fearful:
```

Comment:
please add space after `-`

Reply:
Hey @lmmrssa. Fixed that. Also, I could not get emoji's work in the HTML page. Do I have to remove them?